### PR TITLE
Testing travis ci sauce labs - Ready for Review / Merge @muxator

### PR DESCRIPTION
**Pull request by JohnMcLear**
Suggest doing a squash when merging.

Intentionally commited to ether repo.

- [x] SC latest reference file 404'd.  Used explicit sauce labs version file.
- [x] Attempt to remove wd (this can't be removed)
- [x] Make sure this works for @muxator and @lpagliari -- Okay so Travis runs the tests and these settings are set in Travis -  ``SAUCE_ACCESS_KEY / SAUCE_USERNAME`` -- Email sent to @muxator with credentials 
- [x] Check SL is running OS' we want (See changes to remote_runner.js and now we have more coverage!  This was a nightmare to do(and took hours) though because saucelabs OS / platform / browser names don't match what's required in their own docs [platform configuration tool] for some reason...)  -- I tried to make this future oriented with using 'latest' where possible.
- [x] Make tests that have been broken pass.  I'm not fixing tests that fail due to colibris tho.
- [ ] Understand if tests pass who and how the results are passed - Afaik it returns all tests passed even if some tests fail?!  <-- I suggest merging and doing this as an additional task.  The reason being is that until things are in develop it's hard to go between a working / broken state.

### CODE REVIEW

File: tests/frontend/travis/remote_runner.js [hunk 1]
```diff
@@ -67,41 +67,39 @@ var sauceTestWorker = async.queue(function (testSettings, callback) {
...
-*/
 
-// IE 9
+// 4) IE 10 on Win 8
 sauceTestWorker.push({
-    'platform'       : 'Windows XP'
+    'platform'       : 'Windows 8'
   , 'browserName'    : 'iexplore'
-  , 'version'        : '9'
+  , 'version'        : '10.0'
```
Comment (muxator) [inline tests/frontend/travis/remote_runner.js]: Would it work if we put **IE 11** here? I know it took a lot of time to find these combinations.

IE 10 is practically out of support.

Comment (JohnMcLear) [reply tests/frontend/travis/remote_runner.js]: I intentionally chose IE10 and Edge to get opposite ends of the spectrum from MS.  You could bring another blob in for IE11 should you wish.  We're supposed to have 5 concurrent tests run but afaik it only does 4(saucelabs is weird) so it wouldn't slow things down at all..

Comment (muxator) [reply tests/frontend/travis/remote_runner.js]: Fine. Let's keep IE 10.

**Comment by muxator** [comment]
Pulled in, thanks for the hard work.

Please not I have deleted the original `testing-travis-ci-sauce-labs` in the main repo. A re-clone will be advisable, see https://github.com/ether/etherpad-lite/pull/3745#issuecomment-602875166.

**Comment by JohnMcLear** [comment]
Ack.

**Pull request merged**